frontend/package.json

```diff
@@ -46,7 +46,7 @@
   },
   "devDependencies": {
     "@types/lodash": "^4.17.1",
-    "@types/react": "^18.0.28",
+    "@types/react": "^18.3.3",
```

Comment:
This was me trying to get non-empty react types https://github.com/DefinitelyTyped/DefinitelyTyped/issues/13251#issuecomment-278004160